packages/test-utils/src/externalSigner.ts

```diff
@@ -1,6 +1,6 @@
 import fs from "node:fs";
 import path from "node:path";
-import tmp from "tmp";
+import {dirSync} from "tmp";
```

Comment:
shouldn't we apply same logic as for winstom and system here as well, the "tmp" here is quite informational and indicates that we are just creating temp directory

Reply:
If there were more than one usage of `tmp` I would have preferred that. Bu it's just one function and function name is also clear enough.